addons/sourcemod/scripting/vsh/bosses/boss_announcer.sp

```diff
@@ -274,6 +316,19 @@ methodmap CAnnouncerMinion < SaxtonHaleBase
 		return Plugin_Continue;
 	}
 	
+	public Action OnAttackBuilding(int &victim, int &inflictor, float &damage, int &damagetype, int &weapon, float damageForce[3], float damagePosition[3], int damagecustom)
+	{
+		int iOwner = GetEntPropEnt(victim, Prop_Send, "m_hBuilder");
+		
+		if (TF2_GetClientTeam(iOwner) == TFTeam_Boss)
```

Comment:
Can this ever pass? Don't you want to check if owner is announcer minion about to be changed instead

Reply:
this passes yeah, it's meant to stop minions who are switching from damaging buildings that are on blu
what you suggest is the opposite, but that should be done too